wimm.Secundatives.UnitTests/Extensions/Or_Test.cs

```diff
@@ -0,0 +1,216 @@
+﻿using Xunit;
+
+using wimm.Secundatives.Extensions;
+using System.Threading.Tasks;
+using System;
+using Moq;
+
+namespace wimm.Secundatives.UnitTests.Extensions
+{
+    public class Or_Test
+    {
+        [Fact]
+        public void Or_MaybeFunc_NeitherMaybeExists_ReturnsNone()
+        {
+            Assert.Equal(
```

Comment:
Getting a little aggressive on the newlines aren't we? Not separating this gets to col 85

Reply:
I wrote a long one first then copy pasted so the shape stayed the same. Its all the same to me so I'll reformat